README.md

```diff
@@ -425,6 +425,18 @@ Audited.config do |config|
 end
 ```
 
+#### Changing the table name
+
+If you use a custom audit model _and_ define a different table name inside it, you must mark the `Audited::Audit` superclass as abstract. Otherwise you may get exceptions from attempts to access the missing `audits` table, since ActiveRecord still "sees" a valid, non-abstract model class in `Audited::Audit` and may attempt to access information about its inferred related table.
```

Comment:
Yes, I'm aware this suggests an STI-related solution to a non-STI problem, but in this specific case I think it _does_ make sense; we don't want the base class to be instantiated ever and we definitely want ActiveRecord to use the subclass's table name only.